exchanges/go.mod

```diff
@@ -3,12 +3,11 @@ module github.com/decred/dcrdata/exchanges/v2
 go 1.12
 
 require (
+	decred.org/dcrdex v0.1.1-beta1
```

Comment:
I stupidly made a tag that qualified as semver and we're stuck with it.  Will want to make this go for whatever we have on either the master or release-0.1 branches.  Seeing as I made that dumb tag I think we have no choice but to tag v0.1.1 at the same place as release-v0.1.1, so I'll probably do that.